evaluation.md

```diff
@@ -6,8 +6,59 @@ _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
 **On a scale of 1–10 (where 10 is "a lot"), how much did you have to change your syntax?**
+10. The syntax is not as clear as we wanted it to be initially. Now we're relying
+on one method, `If`, that calls every other method.  Thus, we're passing in strings as
```

Comment:
Our group also had to compromise our syntax because we needed to make it fit some sort of function call.